src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx

```diff
@@ -47,24 +43,22 @@ function QuickbooksOutOfPocketExpenseConfigurationPage({policy}: WithPolicyConne
                             brickRoadIndicator={hasErrors ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
                             shouldShowRightIcon
                             hintText={
-                                reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.VENDOR_BILL && !isLocationEnabled
+                                reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.VENDOR_BILL
                                     ? translate('workspace.qbo.exportVendorBillDescription')
                                     : undefined
                             }
                         />
                     </OfflineWithFeedback>
-                    {!isLocationEnabled && (
-                        <OfflineWithFeedback pendingAction={pendingFields?.reimbursableExpensesAccount}>
-                            <MenuItemWithTopDescription
```

Comment:
This render condition was used so we don't show the account row if the selected entity value is invalid/unsupported (e.g. vendor bill with location enabled).

This is all outdated as the user won't fall into a case where the selected value is invalid.